.markdownlint.jsonc

```diff
@@ -0,0 +1,33 @@
+// Configuration for markdownlint
+// See https://github.com/DavidAnson/markdownlint/blob/v0.32.1/schema/.markdownlint.jsonc
+{
+  
+  // Default state for all rules
+  "default": true,
+
+  // MD013/line-length : Line length : https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md013.md
+  // Justification: Line length standard has not been defined
+  "MD013": false,
+
+  // MD024/no-duplicate-heading : Multiple headings with the same content : https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md024.md
+  // Justification: Some sections have a common template sub-heading format.
+  "MD024": {
+    // Only check sibling headings
+    "siblings_only": true
+  },
+
+  // MD029/ol-prefix : Ordered list item prefix : https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md029.md
+  // Justification: When lists have indented content, it resets the one style lists, and the numbering must be manually set, but then one style can be used from then on.
+  "MD029": false,
+
+  // MD033/no-inline-html : Inline HTML : https://github.com/DavidAnson/markdownlint/blob/v0.32.1/doc/md033.md
+  "MD033": false,
```

Comment:
Is there a reason why we want inline HTML? If not - I would opt for leaving this as default - or at least use the `allowed_elements` option to allowlist certain elements.

Reply:
Removed to be on the safe side if the mkdocs template needed any custom html for any reason. I'll enable as suggested.